crates/ursa-network/src/config.rs

```diff
@@ -82,6 +85,9 @@ impl NetworkConfig {
     fn default_identity() -> String {
         "default".to_string()
     }
+    fn default_kad_replication_factor() -> usize {
+        8
```

Comment:
May I ask why `8` is chosen here? In other words, why you are deviating from the default?

For those interested, the following measurement report on the liveness of Provider Records in the IPFS DHT might be helpful https://github.com/protocol/network-measurements/blob/master/results/rfm17-provider-record-liveness.md.

Reply:
Hi @mxinden, thanks for reaching out, your input is always welcome here :)
We only use kademlia for peer discovery right now, but we are still considering to use it for content routing at some point. The default is set to 8 because that is the desired outbound degree for the meshes. Thanks for sharing the report, I'll definitely have a look!

Reply:
> The default is set to 8 because that is the desired outbound degree for the meshes.

Can you expand on "desired outbound degree for the meshes"? What is an "outbound degree" here? What are you referring to with "meshes"? Sorry in case I am missing something obvious.

The replication factor should (among other things) depend on the expected peer churn. If I publish a value today to X peers, will one of those X peers still be online when someone else queries for that value later on.